src/main/webapp/js/home.js

```diff
@@ -19,7 +19,7 @@ function displayImage() {
     if (this.files && this.files[0]) {
       var img = document.querySelector('img');
       img.src = URL.createObjectURL(this.files[0]);
-      img.onload = imageIsLoaded;
+      //img.onload = imageIsLoaded;
```

Comment:
Just delete the line if you no longer need it.

Reply:
@alyssa-lau This line causes an undefined variable error, and removing it seems not affect the image upload process, is it good to delete this line?

Reply:
Yes, we can delete this line.